src/core/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,85 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { useTheme } from '../utils/hooks/useTheme';
+import '@itwin/itwinui-css/css/side-navigation.css';
+import { CommonProps } from '../utils/props';
+import SvgChevronRight from '@itwin/itwinui-icons-react/cjs/icons/ChevronRight';
+import { IconButton } from '../Buttons';
+
+export type SidebarProps = {
+  /**
+   * Buttons shown in the top portion sidebar.
+   * Recommended to use `SidenavButton`s.
+   */
+  mainItems: React.ReactNode[];
+  /**
+   * Buttons shown at the bottom of the sidebar.
+   */
+  secondaryItems?: React.ReactNode[];
+  /**
+   * Control the visibility of "expander" icon button.
+   * @default 'top'
+   */
+  expanderVisibility?: 'top' | 'bottom' | 'hidden';
+} & Omit<CommonProps, 'title'>;
+
+/**
+ * Describe me here!
+ * @example
+ * Example usages go here!
+ */
+export const Sidebar = (props: SidebarProps) => {
+  const {
+    mainItems,
+    secondaryItems,
+    expanderVisibility = 'top',
+    className,
+    ...rest
+  } = props;
+
+  useTheme();
+
+  const [isExpanded, setIsExpanded] = React.useState(false);
```

Comment:
Should we provide this as a controlled prop too?

Reply:
Yeah, I think we better do.. Maybe there is a reason they want to show it expanded straight away...

Reply:
Added `isExpanded` and `onExpanderClick` props

Reply:
`onExpanderClick` may be unnecessary idk

Reply:
In other components we provide these callbacks, so maybe it is ok to have it here too.

Reply:
The problem is that it's not fully controlled. We change the value of `_isExpanded` (state) even when `isExpanded` (prop) is provided

Reply:
I see that we are doing the same in other components.